doc/upgrade/7.2.md

```diff
@@ -4,9 +4,15 @@ See `doc/bc/changes-7.2.md` for requirements changes and deprecations.
 
 ## MySQL/MariaDB database tables character set change
 
-The character set for MySQL/MariaDB database tables is changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the SQL upgrade statements.
+The character set for MySQL/MariaDB database tables should be changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the statements that shorten table indexes, these must be run first. Beware that these upgrade statements may fail due to index collisions, which may occur when indexes are shortened. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
 
-Beware that these upgrade statements may fail due to index collisions. This is because the indexes have been shortened, so duplicates may occur. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
+After successfully running those statements, change the character set of each table to `utf8mb4`. This can be done as shown below, for each of the standard tables in eZ Platform, and for any custom table you may want to convert. You may specify a collation; if you don't, the default collation for utf8mb4 will be used. We recommend specifying `utf8mb4_unicode_520_ci`, which supports emoji collation (the collation to use depends on your needs).
```

Comment:
Hmm, now I'm gonna nitpick on the last sentence here :grinning: 
Wasn't this supposed to be 
> We recommend specifying `utf8mb4_unicode_520_ci` collation, which supports e.g. emojis

Reply:
@alongosz `utf8mb4_unicode_520_ci` is probably not right for everyone, hence the parentheses remark. Some may want binary, for example.

Reply:
@glye sorry, I meant the part that stands before parentheses. Just updated comment to make it more clear.
I don't get "emoji collation" :)

Reply:
@alongosz Right. I was trying to give a hint while not being too wordy. Any collation "supports" emojis (allows them to be used). By "emoji collation" I mean "is able to tell the difference between different emojis". The standard collation thinks all emojis are the same.

How about:
> We recommend specifying `utf8mb4_unicode_520_ci`, which is able to tell different emojis apart, though you may have other requirements.

Reply:
^ Ok, that sounds good.